resolvers/projectResolver.ts

```diff
@@ -1027,12 +1027,37 @@ export class ProjectResolver {
       .setParameter('slugs', slugs)
       .getMany()
 
+
     const projectsUpdatedListing = projects.map(project => {
       return { id: project.id, listed: listed }
     })
 
+    const segmentProject = await projects.map( async project => {
+      const user = await User.find({ id: Number(project.admin) })
```

Comment:
You can use `User.findOne` instead of `User.find` and it will return one object so you don't have to use `user[0]` you can simply use `user`